examples/jsblocks/index.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html lang="en" data-framework="jsblocks">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<title>jsblocks • TodoMVC</title>
+		<link rel="stylesheet" href="node_modules/todomvc-common/base.css">
+		<link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
+	</head>
+	<body>
+		<section id="todoapp" data-query="view(Todos)" class="todoapp">
```

Comment:
I added the id because the tests were failing without it. I can remove them quickly if this is not a problem.